src/include/platform/NetworkCommissioning.h

```diff
@@ -79,7 +79,7 @@ class Iterator
 struct Network
 {
     uint8_t networkID[kMaxNetworkIDLen];
-    uint8_t networkIDLen;
+    size_t networkIDLen;
```

Comment:
Why? `kMaxNetworkIDLen` is 32, so `uint8_t` is enough.

Reply:
It was to fix a build error from this: https://github.com/project-chip/connectedhomeip/blob/126f6b2915fc26ee17342e1dc80f96f031938a6c/src/platform/Linux/NetworkCommissioningEthernetDriver.cpp#L37

But since we know IF_NAMESIZE will fit in uint8_t, I can static_cast to uint8_t as Darwin does.

Reply:
Done.